code/_onclick/item_attack.dm

```diff
@@ -81,7 +81,8 @@
 	user.do_attack_animation(M)
 	. = M.attacked_by(src, user, def_zone)
 
-	add_attack_logs(user, M, "Attacked with [name] ([uppertext(user.a_intent)]) ([uppertext(damtype)])", (M.ckey && force > 0 && damtype != STAMINA) ? null : ATKLOG_ALMOSTALL)
+	if(force)
+		add_attack_logs(user, M, "Attacked with [name] ([uppertext(user.a_intent)]) ([uppertext(damtype)])", (M.ckey && force > 0 && damtype != STAMINA) ? null : ATKLOG_ALMOSTALL)
```

Comment:
To cut down on `if(force)` checks, I think this should be put somewhere under the `else` on line 73. There, we already know that it has a non-zero force.